index.html

```diff
@@ -375,6 +376,12 @@ <h1>Unitary Fund</h1>
 
     </p>
 
+    <p class="subtitle"><a name="news">News</a></p>
+
+    <p>
+        <a href="news.html">Team intro</a>
```

Comment:
url?

Reply:
I'm wondering if this is not pedantic and shall I just remove it.